lib/hashie/dash.rb

```diff
@@ -174,10 +174,15 @@ def update_attributes!(attributes)
 
     private
 
-    def initialize_attributes(attributes)
+    def initialize_attributes(attributes, options = {})
+      return unless attributes
+
+      set_nils = options.delete(:set_nils)
```

Comment:
This modifies the calling hash, so probably not a great idea?

Reply:
It's an internal function so it's unused data, but I see your point.

Reply:
Since we're purists here, never make assumptions about the caller, things change fast ;)